doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>Replace both links with their abstract page and not to the actual pdfs. (e.g. https://arxiv.org/abs/2006.04528)</p><p> </p><p>method that belong -&gt; method that belongs</p><p>Given a input -&gt; Given an input</p><p>between two instance -&gt; between two instances.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>